drivers/fuel_gauge/sy24561/sy24561.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#define DECI_KELVIN_TO_CELSIUS(temp_dk) (((temp_dk) - 2731) / 10)
+#define CELSIUS_TO_DECI_KELVIN(temp_c)  ((temp_c) * 10 + 2731)
+
+#define PRIdK(_tp)   PRI##_tp ".%" PRI##_tp "K"
+#define dKARGS(_val) (_val / 10), (_val % 10)
```

Comment:
Move it to `sy24561.c`, there is no need to have it in the public interface

Reply:
Sorry didn't saw it, just a tip, if you resolve things, please click on the `resolve conversation` button, with that I know which change requests are resolved or needs further investigation/discussion or whatever.